common/src/main/scala/explore/model/ExploreGridLayouts.scala

```diff
@@ -269,6 +270,40 @@ object ExploreGridLayouts:
       ).withMinWidth
   end observations
 
+  object observationList:
```

Comment:
Because the observation tree on the left is still name `ObsList`, I wonder if this should be named something like `observationSummary` for clarity?

Reply:
`observationSummary` is just the tile, not the view with the tile+plot. I'd rather change `ObsList` to `ObsTree`